code/modules/antagonists/traitor/objectives/destroy_item.dm

```diff
@@ -76,6 +75,10 @@
 	if(length(target_item.special_equipment))
 		special_equipment = target_item.special_equipment
 	replace_in_name("%ITEM%", target_item.name)
+	if(target_item.name == "the Blackbox")
```

Comment:
This seems hacky. Why don't we check for something like the INDESTRUTIBLE resistane flag instead?

Reply:
How do i do that? I'm unfamiliar with checking for flags remotely.

Reply:
```suggestion
	if(target_item.resistance_flags & INDESTRUCTIBLE)
```

Reply:
nvm that won't work, it's not an /obj/item

Reply:
add a variable to `/datum/objective_item/steal` called `destruction_method`.
Set it to null by default. Then, you can check if `target_item.destruction_method != null`. If so, sub in the %METHOD% for whatever is in target_item.destruction_method.

Then you can set `destruction_method = "Too strong to be be destroyed via normal means - needs to be dusted via the supermatter, or burnt in the chapel crematorium."` for the blackbox

Reply:
Ah ok thanks.

Reply:
dang its not compiling gimme a sec